src/components/DatePicker/DatePicker.visual.st.css

```
@@ -7,4 +7,10 @@
     -st-mixin: DatePicker(
         DayFont "{theme: 'Body-M', size: '24px' }",
     );
+}
+
+.selectedDayBorderRadius{
+    -st-mixin: DatePicker(
+            SelectedDayBorderRadius 0,
+    );
```

The indentation looks weird, maybe?
```suggestion
.selectedDayBorderRadius{
    -st-mixin: DatePicker(
       SelectedDayBorderRadius 0,
    );
```